doc/articles/features/fluent-styles.md

```diff
@@ -1,31 +1,36 @@
 # Fluent-styled controls
 
-Uno 3.0 and above supports control styles conforming to the [Fluent design system](https://www.microsoft.com/design/fluent). This article explains how to use them in your app.
+Uno Platform 3.0 and above supports control styles conforming to the [Fluent design system](https://www.microsoft.com/design/fluent). This article explains how to use them in your app.
 
 ## Upgrading existing Uno apps to use Fluent styles
 
-To modify an existing Uno app to use the Fluent control styles, follow these steps:
+Overall, the Uno Platform uses the same mechanism as WinUI 2 to enable Fluent styles. After installing the `Uno.UI` Nuget version 3.0 or above Fluent styles are enabled by specifying the `XamlControlsResources` within the application's resources (inside `App.xaml`).
 
-1. Update `Uno.UI` NuGet packages to 3.0 or above.
-1. In the `UWP` head project of your solution, install the [WinUI 2 NuGet package](https://www.nuget.org/packages/Microsoft.UI.Xaml).
-1. Add the `XamlControlsResources` resource dictionary to your application resources in `App.xaml`:
+For all platform heads except UWP Fluent design styles will then be used by default within your app. However, for the UWP head, an additional WinUI 2 Nuget package reference must be added. This is following the same process as UWP because, for the UWP head, the Uno Platform is not used.
```

Comment:
```suggestion
For the UWP head, an additional WinUI 2 Nuget package reference must be added. This is following the same process as UWP because, for the UWP head, the Uno Platform is not used.
```

Reply:
I worded it like - emphasizing other heads are ready without the Nuget - for those like me that might not even have a UWP head in the Uno solution. (UWP solution is separate and maintains original structure for compatibility). I can certainly change it if you still think it's best not to emphasize this.

Reply:
I realize that the font still needs to be specified for 'Fluent design styles... to be used by default' and work correctly. However, that isn't mentioned at this point in the document. Since you also added an emphasis below about the UWP head potentially not being present I'll remove this section as you recommend.